### DISCUSSION TITLE
data-toggle="collapse"   Can I hover and expand the mouse?

### DISCUSSION THREAD

**Discussion by bbhxwl** [twbs/bootstrap#35189] [Q&A]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

data-toggle="collapse"   Can I hover and expand the mouse?

### Motivation and context

data-toggle="collapse"   Can I hover and expand the mouse?

**Comment by CyrilKrylatov** [comment]
Hi,

I think you may need to use some JavaScript to do that with Collapse's methods https://getbootstrap.com/docs/5.1/components/collapse/#methods and do something like
```js
new bootstrap.Collapse(myCollapse, {
  show: true
})
```

on a mouseenter event on the `myCollapse` event.

For my very own point of view, Bootstrap should not introduce that behavior since users don't know / didn't ask that this element will expand when hovering. Could be really bothering for them.

**Reply by benkoshy** [reply]
I wonder if it would be benefit to users if the above Javascript could be done declaratively: e.g.

`data-toggle="hover-collapse"` or something like that?
